### ISSUE TITLE
Plausible Deniability

### ISSUE THREAD

**Issue by eragmus** [voisine/breadwallet#278]
Please read this article:
> http://dafacto.com/plausible-deniability-and-bitcoin-wallets/

This idea seems almost mandatory to implement, since it would do much to boost security. It's especially true if the user is able to create as many 'accounts' as wanted, to further increase the plausible deniability aspect. It would be nice even in non-threatening yet still somewhat 'iffy' situations, such as showing a friend how bitcoin works. Instead of having to open up your main wallet and create an awkward situation by hiding your wallet passcode as you enter it, you could confidently just open up a smaller wallet with $5 worth or something.

This idea may also mesh well with the earlier "multi-account" idea; in fact it could be a security-enhancing side of it, in addition to the privacy-enhancing side (.. of separating 'main' account's txs from 'daily' account txs):
> https://github.com/voisine/breadwallet/issues/274

**Comment by voisine** [comment]
This was seriously considered by the banking industry when ATMs were being introduced. Each user would have a panic pin which would display fake account info, it was ultimately rejected due to being too confusing for users, that attackers would be aware of it, and that users wouldn't have the presence of mind to use it when under duress.

Bitcoin of course is a different technology than ATMs with different risks and use profiles, so we shouldn't accept their conclusions without consideration, but the difficulties are still largely applicable.

**Comment by voisine** [comment]
Might a better solution be to have a multi-sig service that provides theft insurance? Relying on users to correctly handle themselves in a scenario like that seems like it would have a relatively low success rate. Regular people are never going to behave like security experts.

**Comment by eragmus** [comment]
Yeah, it's tricky. I actually made a similar argument on that article in the comments, and the writer commented back:
> "Well, that's the whole idea behind plausible deniability — they might suspect you have other wallets, but the can't know. Want extra security, have a low and a medium balance wallet."

I'm not 100% convinced on the practicality of it, but at the minimum, we can say that 'advanced' or 'pro' users will surely take advantage of the feature. The others -- yeah, I agree with you that I can't really expect too much. I wonder what published studies say exactly about "presence of mind to use it when under duress". Maybe the implementation can be fixed.

>> Might a better solution be to have a multi-sig service that provides theft insurance?

Theft insurance? Does this mean the user pays a monthly fee to insure the balance, and if stolen, is fully compensated? Or does this mean that multisig is employed to prevent the user from spending freely on a unilateral basis (and that this restricted spending ability makes the 'theft insurance' feasible)?

If the former, a monthly fee sounds unappealing. If the latter, it may help with mainstream adoption, but 'power' users may not like to give up control to a 'trusted third party' (TTP). What if the TTP gets hacked or coerced or such? I assume there's no impact on speed of making transactions with this route; I'll guess it's automated somehow.

**Comment by voisine** [comment]
A multi-sig provider is in an excellent position to detect and prevent fraud/theft transactions. The stronger their confidence in their product, the cheaper they should be able to offer to insure the funds. Users don't have to give up control, the service provider only has the power to deny transactions from the mobile. If the user disagrees, they would have a backup key kept somewhere safe or by another party like with a lawyer or safe deposit box. The insurer might offer discounts based on the level of security given to the storage of the backup key.

**Comment by voisine** [comment]
Also if the amount stolen is high enough, the insurer has an incentive to hunt down and extract repayment from the perpetrator, ala the Pinkertons.
